src/app.js

```diff
@@ -1,12 +1,88 @@
+import axios from "axios";
+import distance from "euclidean-distance";
+
+// Get the API token
+async function getToken() {
+  let token;
+  try {
+    const response = await axios.post(
+      "https://blue-bottle-api-test.herokuapp.com/v1/tokens",
```

Comment:
How will you handle different paths for the api, lets say that the staging api will be `https://blue-bottle-api.staging.herokuapp.com`

Reply:
I can improve my API URL handling by using a config file for the base URL. Even it could be an environment variable if it is going to be run under different environments (like production, staging, etc).